spec/features/instant_sessions_spec.rb

```diff
@@ -9,7 +9,7 @@
     allow(CfeConnection).to receive(:assess).and_return(api_response)
   end
 
-  context "without conditional reveals for assets enabled", :legacy_assets_no_reveal do
+  context "without conditional reveals for assets enabled" do
```

Comment:
It feels strange to keep this test given the 'context' wording. Is the wording wrong, or should the test be deleted?